Source/LightboxImage.swift

```diff
@@ -26,7 +26,7 @@ open class LightboxImage {
       imageView.image = image
       completion?(image)
     } else if let imageURL = imageURL {
-      LightboxConfig.loadImage(imageView, imageURL) { [weak self] error, image in
+      LightboxConfig.loadImage(imageView, imageURL) { [weak self] _, image in
```

Comment:
I'd like to draw your attention to the `error` variable here.
I wanted to do something with it, such as printing it or processing it, but I lack of any relevant ideas. Can we do something about it or is it okay to ignore it in this scope?
Thanks!

Reply:
The error should probably be passed to the completion so that the user has the chance of knowing what went wrong with loading the remote image.

Reply:
I think you can open an issue to improve the completion of `addImageTo` completion. It feels beyond scope for this PR.

Reply:
I feel the same about the outOfScope exception.
I'll open this issue for enhancement soon.
Thanks for your feedback.